apps/rush-lib/src/api/EnvironmentConfiguration.ts

```diff
@@ -89,7 +89,12 @@ export const enum EnvironmentVariableNames {
    *
    * POSIX is a registered trademark of the Institute of Electrical and Electronic Engineers, Inc.
    */
-  RUSH_GLOBAL_FOLDER = 'RUSH_GLOBAL_FOLDER'
+  RUSH_GLOBAL_FOLDER = 'RUSH_GLOBAL_FOLDER',
+
+  /**
+   * Provides the connection string for a remote build cache, if configured.
+   */
+  RUSH_BUILD_CACHE_CONNECTION_STRING = 'RUSH_BUILD_CACHE_CONNECTION_STRING'
```

Comment:
The docs should:
- Summarize the syntax
- Show a realistic example value
- Provide a link to the full specification for the syntax

If the value has incorrect syntax, we should check that Rush displays a helpful error message.

Reply:
Let's replace this with just a "Credential," so we don't have to deal with a connection string at all.

In the Azure Storage case, this will just store a SAS.